hugegraph-core/pom.xml

```diff
@@ -13,6 +13,22 @@
 
     <properties>
         <top.level.dir>${basedir}/..</top.level.dir>
+        <jraft.version>1.3.11</jraft.version>
+        <ohc.version>0.7.4</ohc.version>
+        <lz4.version>1.8.0</lz4.version>
+        <apdplat-word.version>1.3.1</apdplat-word.version>
+        <lucene-analyzers-smartcn.version>8.11.2</lucene-analyzers-smartcn.version>
+        <mmseg4j-core.version>1.10.0</mmseg4j-core.version>
+        <jcseg.version>2.6.2</jcseg.version>
+        <hanlp.version>portable-1.8.3</hanlp.version>
+        <ansj-seg.version>5.1.6</ansj-seg.version>
+        <lucene-core.version>8.11.2</lucene-core.version>
+        <jieba-analysis.version>1.0.2</jieba-analysis.version>
+        <ikanalyzer.version>2012_u6</ikanalyzer.version>
+        <commons-compress.version>1.21</commons-compress.version>
+        <eclipse-collections.version>11.1.0</eclipse-collections.version>
+        <fastutil.version>8.5.9</fastutil.version>
+        <jjwt.version>0.11.5</jjwt.version>
```

Comment:
can you highlight which versions are changed? or it's ok to just leave some comments in the reply

Reply:
jraft 1.3.9->1.3.11
ohc-core 0.7.0->0.7.4
org.apdplat.word 1.3->1.3.1
hanlp portable-1.5.0->portable-1.8.3
lucene-analyzers-smartcn,lucene-core 7.4.0->8.11.2
jcseg-core->2.2.0 ->2.6.2
lz4-java 1.7.1->1.8.0
eclipse-collections 10.4.0->11.1.0
fastutil 8.1.0->8.4.0
jjwt 0.11.2->0.11.5